sensor_msgs/JointState[] poses
---
trajectory_msgs/JointTrajectory joint_trajectory
string error

